src/components/ModalSendTx.js

```diff
@@ -81,7 +81,8 @@ export class ModalSendTx extends React.Component {
     // If we are using the wallet service facade, we should avail of the validated PIN
     // to renew the auth token.
     if (this.props.useWalletService) {
-      await this.props.wallet.validateAndRenewAuthToken(pin);
+      const wallet = getGlobalWallet();
+      await wallet.validateAndRenewAuthToken(pin);
```

Comment:
```suggestion
      const { validateAndRenewAuthToken } = getGlobalWallet();
      await validateAndRenewAuthToken(pin);
```

Reply:
It's not possible to use this syntax with a HathorWallet method: it crashes the application.